adapters/eplanning/eplanning.go

```diff
@@ -0,0 +1,147 @@
+package eplanning
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type EPlanningAdapter struct {
+	http *adapters.HTTPAdapter
+	URI  string
+}
+
+type EPlanningRequest struct {
+	id      string
+	user    *openrtb.User
+	device  *openrtb.Device
+	adUnits []*EPlanningAdUnit
+}
+
+type EPlanningBid struct {
+	Id     string  `json:"id"`
+	Price  float64 `json:"price,omitempty"`
+	Width  uint64  `json:"w,omitempty"`
+	Height uint64  `json:"h,omitempty"`
+	DealId string  `json:"dealid,omitempty"`
+}
+
+type EPlanningAdUnit struct {
+	Id         string
+	Bidfloor   float64
+	Instl      int8
+	SspSpaceId int `json:"ssp_espacio_id,omitempty"`
+	Video      *openrtb.Video
+	Banner     *openrtb.Banner
+}
+
+func (adapter *EPlanningAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters.RequestData, []error) {
+	ePlanningRequest, errors := openRtbToEPlanningRequest(request)
+	if len(ePlanningRequest.adUnits) == 0 {
+		return nil, errors
+	}
+
+	reqJSON, err := json.Marshal(request)
```

Comment:
I think you want to marshal your `ePlanningRequest` here, not the initial request that was passed to you. Otherwise you don't seem to be using the generated `ePlanningRequest` for anything?